contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,356 @@
+within an issue can be challenging. The flat hierarchy of issues can also make
+navigation and categorization tricky. While not all community members might
+not be comfortable using Git directly, it is imperative that as a community we
+work to educate people on a standard set of tools so they can take their
+experience to other projects they may decide to work on in the future. While
+git is a fantastic version control system (VCS), it is not a project management
+tool nor a cogent way of managing an architectural catalog or backlog; this
+proposal is limited to motivating the creation of a standardized definition of
+work in order to facilitate project management. This primitive for describing
+a unit of work may also allow contributors to create their own personalized
+view of the state of the project while relying on Git and GitHub for consistency
+and durable storage.
+
+Ideally release notes should [tell a story][] which
+is compelling enough to encourage users and operators to upgrade their clusters.
+Without a standardized mechanism for describing important enhancements our
+talented technical writers and product managers struggle to weave a coherent
+narrative explaining why a particular release is important. Additionally for
+critical infrastructure such as Kubernetes adopters need a forward looking road map
+in order to plan their adoption strategy.
+
+[tell a story]: https://blog.rust-lang.org/2017/08/31/Rust-1.20.html 
+[road to Go 2]: https://blog.golang.org/toward-go2
+[survey data]: http://opensourcesurvey.org/2017/
+
+## Detailed design
+
+### What type of work should be tracked by a KEP
+
+The definition of what constitutes an "enhancement" is a foundational concern
+for the Kubernetes project. Roughly any Kubernetes user or operator facing
+enhancement should follow the KEP process: if an enhancement would be described
+in either written or verbal communication to anyone besides the KEP author or
+developer then consider creating a KEP. One concrete example is an enhancement
+which should be communicated to SIG Release or SIG PM.
+
+Without detailed information explaining the motivation for an enhancement SIGs
+must first approve a proposal, agreeing to a motivation over a mailing list,
+video call, or hallway conversation. During the release process this motivation
+must be rediscovered by the SIG, hopefully by finding a design proposal. The
+process of announcing an enhancement through release notes suggests another
+heuristic for describing what work should be tracked through an KEP: anything
+that would require a design proposal. In fact it is possible to consider a KEP
+an enhancement to the design proposal process in which design proposals are
+used throughout the process of proposing an enhancement, scoping its design,
+tracking its implementation, and agreeing on criteria for graduation to general
+availability.
+
+As the local bodies of governance SIGs should have broad latitude in describing
```

Comment:
We're going to need some criteria. For instance, every API change should have a proposal.

Reply:
nit: Comma after 'governance'.

Reply:
@bgrant0607, I think the guidance belongs in each SIG charter since ultimately the SIG should have the latitude to decide what their own development process is

Reply:
@calebamiles:
> @bgrant0607, I think the guidance belongs in each SIG charter since ultimately the SIG should have the latitude to decide what their own development process is

still need some guidance for things like API changes, or for efforts that span sigs

Reply:
Doesn't that contradict the first paragraph of this subsection?